### PR TITLE
Moved RunServer to edu.umass.cs.gnsserver.utils/RunCommand.

### PR DESCRIPTION
**Pull request by westyUMass**
Also removed some warnings.

### CODE REVIEW

File: test/edu/umass/cs/gnsclient/client/integrationtests/ServerConnectTest.java [hunk 1]
```diff
@@ -234,20 +213,20 @@ public static void setUpBeforeClass() throws Exception {
...
 	System.out.println("Waiting for servers to be ready...");
-	output = RunServer.command("cat " + gpConfFile + " | grep \"reconfigurator\\.\" | wc -l ", ".", false);
+	output = RunCommand.command("cat " + gpConfFile + " | grep \"reconfigurator\\.\" | wc -l ", ".", false);
 	int numRC = Integer.parseInt(output.get(0));
-	output = RunServer.command("cat " + gpConfFile + " | grep \"active\\.\" | wc -l ", ".", false);
+	output = RunCommand.command("cat " + gpConfFile + " | grep \"active\\.\" | wc -l ", ".", false);
 	int numAR = Integer.parseInt(output.get(0));
 	int numServers = numRC + numAR;
-	output = RunServer.command("cat " + logFiles + " | grep \"server ready\" | wc -l ", ".", false);
+	output = RunCommand.command("cat " + logFiles + " | grep \"server ready\" | wc -l ", ".", false);
```
Comment (akarthik10) [inline test/edu/umass/cs/gnsclient/client/integrationtests/ServerConnectTest.java]: This [may fail](http://unix.stackexchange.com/questions/19907/what-makes-grep-consider-a-file-to-be-binary) if the log file contains a NUL character. Turning on FINEST logging for NIO will put NUL characters in log files. Using `grep -a` will avoid this. (See `ServerIntegrationTest.java`)

File: test/edu/umass/cs/gnsclient/client/integrationtests/ServerConnectTest.java [hunk 2]
```diff
@@ -234,20 +213,20 @@ public static void setUpBeforeClass() throws Exception {
...
 	int numServersUp = Integer.parseInt(output.get(0));
 	while (numServersUp < numServers){
     		Thread.sleep(5000);
-		output = RunServer.command("cat " + logFiles + " | grep \"server ready\" | wc -l ", ".", false);
+		output = RunCommand.command("cat " + logFiles + " | grep \"server ready\" | wc -l ", ".", false);
```
Comment (akarthik10) [inline test/edu/umass/cs/gnsclient/client/integrationtests/ServerConnectTest.java]: Same as above.

File: test/edu/umass/cs/gnsclient/client/integrationtests/ServerConnectTest.java (hunk 1 above)
Comment (westyUMass) [inline test/edu/umass/cs/gnsclient/client/integrationtests/ServerConnectTest.java]: OK. I didn't write that code so feel free to change it. I'm not even sure what ServerConnectTest is or who wrote it or if it's still in use.

**Comment by westyUMass** [comment]
Thanks.

**Pull request merged**